### PR TITLE
Module initialization

### PR DESCRIPTION
**Pull request by confusdcodr**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -0,0 +1,35 @@
+variable "create_config_authorization" {
+  description = "Toggle that controls creation/management of a config authorization"
+  type        = "string"
+  default     = "false"
```
Comment (userhas404d) [inline variables.tf]: Just a consistency thing but our other modules have declared booleans as just 
```
variable "some_check" {
  description = "toggle"
  default = false
}
```

File: variables.tf [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+variable "create_config_aggregator" {
+  description = "Toggle that controls creation/management of a config aggregator"
+  type        = "string"
+  default     = "false"
```
Comment (userhas404d) [inline variables.tf]: same here

Comment (confusdcodr) [reply variables.tf]: Fixed

File: variables.tf [hunk 3]
```diff
@@ -0,0 +1,35 @@
+variable "create_config_authorization" {
+  description = "Toggle that controls creation/management of a config authorization"
+  type        = "string"
```
Comment (userhas404d) [inline variables.tf]: I think `type = "string"` should be removed as well.

File: variables.tf [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  default     = false
+}
+
+variable "create_config_aggregator" {
+  description = "Toggle that controls creation/management of a config aggregator"
+  type        = "string"
```
Comment (userhas404d) [inline variables.tf]: same here

File: variables.tf (hunk 3 above)
Comment (confusdcodr) [reply variables.tf]: Fixed

**Pull request merged**